package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "eth-components",
-  "version": "2.2.1",
+  "version": "2.3.0",
```

Comment:
Suggestion of version change, make sense that way?

Reply:
yes